### ISSUE TITLE
Use minified userscript for extension

### ISSUE THREAD

**Issue by qsniyg** [qsniyg/maxurl#832]
(Thanks to Liz on discord for the idea.)

This would allow for some performance improvements.

The downside is that users wouldn't be able to read the userscript if opening or debugging the addon. Perhaps sourcemapping could help fix this?

**Comment by MinuteAd8502** [comment]
I don't think that is needed. HTTP Compression (gzip, br) is almost same. It increases download speed and reduces size.
As for script execution speed I don't know, did not compare.

**Comment by qsniyg** [comment]
Sorry, I should have clarified better. This is for the packaged extension version. The only reason for it would be to improve the initial compile time for the script. However, if the browser has already cached it (which, in theory, it should), this should be a negligible improvement, hence why I'm a bit reluctant to add it in.
